pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -117,11 +125,20 @@ func validateThresholds(thresholds api.ResourceThresholds) bool {
 	for name := range thresholds {
 		switch name {
 		case v1.ResourceCPU:
-			continue
+			if !validateResourcePercentage(thresholds[v1.ResourceCPU]) {
```

Comment:
Can you refactor the function to return `fmt.Errorf("cpu threshold not valid")` instead of bool? Logging Info/Error and return bool is not practical. Instead, called of `validateThresholds` can decide what to do about non-nil error.

Reply:
done